### PR TITLE
README.txt: add additional details for installing a full image.

### PR DESCRIPTION
**Pull request by pebenito**

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -35,26 +35,46 @@ found at
...
-will need to make a few more changes. Assuming you are in safe mode and
-the run mode filesystem is mounted at /mnt/userfs:
+more packages of NI's own software.
+
+** NOTE **
+Building packages through OpenEmbedded can use significant disk space,
+on the order of tens of gigabytes. If you are preparing a virtual machine
+to build images, make sure to allocate sufficient disk space.
+
+The kernel image created by the build system cannot be used.  If you
```
Comment (bmouring) [inline README.txt]: This is not strictly correct for all hardware. The kernel would be usable from an x64 build. Adjust the verbiage.

File: README.txt [hunk 2]
```diff
@@ -35,26 +35,46 @@ found at
...
+
+    nisystemformat -f -t ubifs
+
+4. Unpack the rootfs tar.bz2 in the run mode filesystem directory,
+   typically /mnt/userfs.
+
+5. Restore the NI-built kernel and modules to their original locations
+   in /boot and /lib/modules or install your custom built
+   kernel, as described in KERNEL_SOURCE.txt.
+
```
Comment (bmouring) [inline README.txt]: Indent this list to distinguish it as a separate list

File: README.txt [hunk 3]
```diff
@@ -35,26 +35,46 @@ found at
...
+to build images, make sure to allocate sufficient disk space.
+
+The kernel image created by the build system cannot be used.  If you
+will be compiling a custom kernel, see github.com/ni/linux for the
+kernel source.  See KERNEL_SOURCE.txt in this directory for instructions
+to build the kernel.
+
+To install a complete image onto the target:
+
+1. Boot into safe mode.
```
Comment (bmouring) [inline README.txt]: + Refer to the hardware manual for instructions on how to do this.

**Comment by bmouring** [comment]
Thanks for the update. I've added a few bits of feedback.

File: README.txt (hunk 3 above)
Comment (bmouring) [inline README.txt]: Somehow, my clever patch plussign got lost. Basically, this should be added to the first step for deploying an image.

**Comment by pebenito** [comment]
Do you prefer additional commits or replace the existing commit?

**Comment by bmouring** [comment]
Either way, doesn't really make much difference to me.

**Comment by bmouring** [comment]
Actually, you should rework the commit, make sure that the commit message has your Signed-off-by on it

**Comment by pebenito** [comment]
Updated.

**Pull request merged**